aws-athena/views/default-vw_pin_history.sql

```diff
@@ -2,15 +2,18 @@
 SELECT
     vwpv.pin,
     vwpv.year,
-    par.class,
+    REGEXP_REPLACE(par.class, '[^[:alnum:]]', '') AS class,
```

Comment:
There *are* classes in pardat with hyphens in them. This new filtering should avoid stripping out A & B suffixes from our class codes.

Reply:
**question:** Are other class fields in other views treated the same way? If not, we need to standardize this across views.

Reply:
Yes, we will need to update class code cleaning in other views. I've already made an [issue](https://github.com/ccao-data/data-architecture/issues/341) for it.